### ISSUE TITLE
packaging...

### ISSUE THREAD

**Issue by mobinmob** [mobinmob/void-66-services#47]
A single package will be created for all services - I will add it to the [boot-66serv PR](https://github.com/void-linux/void-packages/pull/25743). The changes in the Makefile and removal of `@VERSION@` make the process easy. My current template is the following:

```
# Template file for 'void-66-services'
pkgname=void-66-services
version=0.0.1
revision=1
_version=f0553f75e7380c15df30075e7bad9ecb7f95a8dc
wrksrc="${pkgname}-${_version}"
build_style=gnu-makefile
depends="66"
short_desc="66 service frontend files for voidlinux"
maintainer="mobinmob <mobinmob@disroot.org>"
license="BSD-2-Clause"
homepage="https://github.com/mobinmob/void-66-services/"
distfiles="https://github.com/mobinmob/void-66-services/archive/${_version}.tar.gz"
checksum=333d807ea0a91bf31b6a5cebf96363ff33450c0623597a1552efcf59c6bbddbe

post_install() {
        vlicense LICENSE
        vlicense AUTHORS
}
```

**Comment by mobinmob** [comment]
0.0.1 is not available and the template is in https://github.com/void-linux/void-packages/pull/25743. A template to package master and devel branches will be included in the repo.

**Comment by mobinmob** [comment]
Closing, templates for master and devel branches are under packaging/ - released in version 0.0.2.
